demo/README.md

```diff
@@ -86,5 +86,6 @@ Alternatively, a filepath to the cert file can also be directly passed via the `
 ```
 ansible-playbook demo/create.yml \
     --extra-vars "flightctl_host='your-service-address-here'" \
-    --extra-vars "flightctl_ca_path=path/to/cert/ca.crt"
+    --extra-vars "flightctl_ca_path=path/to/cert/ca.crt" \
+    --extra-vars "flightctl_validate_certs=True"
```

Comment:
question: is validate certs the default?

Reply:
Yes the collection does default to True for cert validation but the demo playbooks defaulted to False, changed these demos to do the same as that seems like a better standard